ai21/models/responses/assistant_response.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+from typing import Optional, List, Literal, TypedDict
+
+from ai21.models.ai21_base_model import AI21BaseModel
+
+
+Optimization = Literal["cost", "latency"]
+Model = Literal["jamba-1-5", "jamba-1-5-large"]
```

Comment:
I'd remove `models` and keep it `str` because they would require extra maintenance